docs/reference/eql/syntax.asciidoc

```diff
@@ -127,14 +127,19 @@ to the right. Otherwise returns `false`.
 Returns `true` if the value to the left of the operator is less than or equal to
 the value to the right. Otherwise returns `false`.
 
-`==` (equal)::
+`==` (equal, case-sensitive)::
 Returns `true` if the values to the left and right of the operator are equal.
-Otherwise returns `false`.
-+
+Otherwise returns `false`. For strings, matching is case-sensitive.
+
+`:` (equal, case-insensitive)::
+Returns `true` if strings to the left and right of the operator are equal.
+Otherwise returns `false`. Matching is case-insensitive and can only be used to
+compare strings. Wildcard (`*`) patterns are not supported.
```

Comment:
> Wildcard (`*`) patterns are not supported.

It looks like the operator currently supports these. Let me know if this should be removed.